.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: android
+android:
+  components:
+  - build-tools-28.0.3
+  - android-28
+script:
+- chmod +x ./gradlew
+- ./gradlew build
+#- ./gradlew build connectedCheck
```

Comment:
Do we need commented code?

Reply:
This line shows that `connectedCheck` isn't called, so tests aren't run. Now we haven't any tests so I decided to not spend time with test settings like an Android virtual device deploying.

To sum up, when tests start to be required, we will uncomment the ninth line and remove the eighth one. Also, we will google something like `travis connectedCheck` and apply additional settings to the script.

Reply:
I've just added a similar comment there.

Reply:
Ok ship it then :)